sdk/src/transaction.rs

```diff
@@ -789,4 +814,103 @@ mod tests {
             vec![Signature::default(), Signature::default()]
         );
     }
+
+    fn new_secp256k1_instruction(
```

Comment:
Can this live in the program, for consistency's sake?

Reply:
yea.. have to duplicate the dependencies on libsecp256k1/sha3/digest from sdk though.